pcsx2/CDVD/ChdFileReader.cpp

```diff
@@ -0,0 +1,173 @@
+#include "PrecompiledHeader.h"
```

Comment:
Missing copyright header and need to run clang format on the file.

Reply:
Fixed in last commit.